catroid/src/main/java/org/catrobat/catroid/content/bricks/PlaceAtBrick.java

```diff
@@ -65,4 +133,19 @@ public int getViewResource() {
 				getFormulaWithBrickField(BrickField.Y_POSITION)));
 		return null;
 	}
+
+	@Override
+	public void saveNewCoordinates(int x, int y) {
+		setFormulaWithBrickField(BrickField.X_POSITION, new Formula(x));
+		setFormulaWithBrickField(BrickField.Y_POSITION, new Formula(y));
+	}
+
+	@Override
+	public int describeContents() {
+		return 0;
+	}
+
+	@Override
```

Comment:
why does VisualPlacementListener have to be a parcelable if the parcelable methods are just empty overrides?